src/components/GasFeeWarning.test.tsx

```diff
@@ -0,0 +1,161 @@
+    ${'depositing max amount of CELO'}            | ${GasFeeWarningFlow.Deposit}  | ${mockPreparedTransactionNeedDecreaseCelo} | ${'CELO'}         | ${'gasFeeWarning.title, {"tokenSymbol":"CELO"}'} | ${'gasFeeWarning.descriptionMaxAmount, {"action":"gasFeeWarning.actions.depositing","tokenSymbol":"CELO"}'}     | ${'gasFeeWarning.ctaGasToken, {"verb":"gasFeeWarning.verb.deposit"}'}
+    ${'depositing max amount of ETH'}             | ${GasFeeWarningFlow.Deposit}  | ${mockPreparedTransactionNeedDecreaseEth}  | ${'ETH'}          | ${'gasFeeWarning.title, {"tokenSymbol":"ETH"}'}  | ${'gasFeeWarning.descriptionMaxAmount, {"action":"gasFeeWarning.actions.depositing","tokenSymbol":"ETH"}'}      | ${'gasFeeWarning.ctaGasToken, {"verb":"gasFeeWarning.verb.deposit"}'}
+    ${'depositing with insufficient CELO'}        | ${GasFeeWarningFlow.Deposit}  | ${mockPreparedTransactionNotEnoughCelo}    | ${'CELO'}         | ${'gasFeeWarning.title, {"tokenSymbol":"CELO"}'} | ${'gasFeeWarning.descriptionNotEnoughGas, {"action":"gasFeeWarning.actions.depositing","tokenSymbol":"CELO"}'}  | ${'gasFeeWarning.cta, {"tokenSymbol":"CELO"}'}
+    ${'depositing with insufficient ETH'}         | ${GasFeeWarningFlow.Deposit}  | ${mockPreparedTransactionNotEnoughEth}     | ${'ETH'}          | ${'gasFeeWarning.title, {"tokenSymbol":"ETH"}'}  | ${'gasFeeWarning.descriptionNotEnoughGas, {"action":"gasFeeWarning.actions.depositing","tokenSymbol":"ETH"}'}   | ${'gasFeeWarning.cta, {"tokenSymbol":"ETH"}'}
+    ${'dapp transaction with max amount of CELO'} | ${GasFeeWarningFlow.Dapp}     | ${mockPreparedTransactionNeedDecreaseCelo} | ${'CELO'}         | ${'gasFeeWarning.titleDapp'}                     | ${'gasFeeWarning.descriptionDapp, {"tokenSymbol":"CELO"}'}                                                      | ${undefined}
+    ${'dapp transaction with max amount of ETH'}  | ${GasFeeWarningFlow.Dapp}     | ${mockPreparedTransactionNeedDecreaseEth}  | ${'ETH'}          | ${'gasFeeWarning.titleDapp'}                     | ${'gasFeeWarning.descriptionDapp, {"tokenSymbol":"ETH"}'}                                                       | ${undefined}
+    ${'dapp transaction with insufficient CELO'}  | ${GasFeeWarningFlow.Dapp}     | ${mockPreparedTransactionNotEnoughCelo}    | ${'CELO'}         | ${'gasFeeWarning.titleDapp'}                     | ${'gasFeeWarning.descriptionDapp, {"tokenSymbol":"CELO"}'}                                                      | ${undefined}
+    ${'dapp transaction with insufficient ETH'}   | ${GasFeeWarningFlow.Dapp}     | ${mockPreparedTransactionNotEnoughEth}     | ${'ETH'}          | ${'gasFeeWarning.titleDapp'}                     | ${'gasFeeWarning.descriptionDapp, {"tokenSymbol":"ETH"}'}                                                       | ${undefined}
```

Comment:
nit: these test cases are a bit hard to read. It'd be nice to break up with comments.

```TypeScript
    scenario                                      | flow                          | prepareTransactionsResult                  | feeCurrencySymbol | title                                            | description                                                                                                     | ctaLabel
    // SENDING CASES
    ${'sending max amount of CELO'}               | ${GasFeeWarningFlow.Send}     | ${mockPreparedTransactionNeedDecreaseCelo} | ${'CELO'}         | ${'gasFeeWarning.title, {"tokenSymbol":"CELO"}'} | ${'gasFeeWarning.descriptionMaxAmount, {"action":"gasFeeWarning.actions.sending","tokenSymbol":"CELO"}'}        | ${'gasFeeWarning.ctaGasToken, {"verb":"gasFeeWarning.verb.send"}'}
    ...
    // SWAPPING CASES
    ${'swapping max amount of CELO'}              | ${GasFeeWarningFlow.Swap}     | ${mockPreparedTransactionNeedDecreaseCelo} | ${'CELO'}         | ${'gasFeeWarning.title, {"tokenSymbol":"CELO"}'} | ${'gasFeeWarning.descriptionMaxAmount, {"action":"gasFeeWarning.actions.swapping","tokenSymbol":"CELO"}'}       | ${'gasFeeWarning.ctaGasToken, {"verb":"gasFeeWarning.verb.swap"}'}
    ...
    // WITHDRAWING CASES
    ${'withdrawing max amount of CELO'}           | ${GasFeeWarningFlow.Withdraw} | ${mockPreparedTransactionNeedDecreaseCelo} | ${'CELO'}         | ${'gasFeeWarning.title, {"tokenSymbol":"CELO"}'} | ${'gasFeeWarning.descriptionMaxAmount, {"action":"gasFeeWarning.actions.withdrawing","tokenSymbol":"CELO"}'}    | ${'gasFeeWarning.ctaGasToken, {"verb":"gasFeeWarning.verb.withdraw"}'}
    ...
    // DEPOSITING CASES
    ${'depositing max amount of CELO'}            | ${GasFeeWarningFlow.Deposit}  | ${mockPreparedTransactionNeedDecreaseCelo} | ${'CELO'}         | ${'gasFeeWarning.title, {"tokenSymbol":"CELO"}'} | ${'gasFeeWarning.descriptionMaxAmount, {"action":"gasFeeWarning.actions.depositing","tokenSymbol":"CELO"}'}     | ${'gasFeeWarning.ctaGasToken, {"verb":"gasFeeWarning.verb.deposit"}'}
    ...
    // DAPP CASES
    ${'dapp transaction with max amount of CELO'} | ${GasFeeWarningFlow.Dapp}     | ${mockPreparedTransactionNeedDecreaseCelo} | ${'CELO'}         | ${'gasFeeWarning.titleDapp'}                     | ${'gasFeeWarning.descriptionDapp, {"tokenSymbol":"CELO"}'}                                                      | ${undefined}
    ...

Reply:
added in [7a33d2b](https://github.com/valora-inc/wallet/pull/6385/commits/7a33d2b785245904a61f8f91870a3334c52aa935)

Reply:
ahh actually prettier doesn't allow this, it removed them except for the first one, so I am leaving them out unless there is a different way to comment like this

Reply:
You can add a `// prettier-ignore` above the test case to prevent prettier from stripping out the comments, but then the test below isn't formatted by prettier. I am fine with leaving the comments out also.